updatens.py

```diff
@@ -0,0 +1,151 @@
+#! /usr/env/python3
+# requires requests, dnspython
+import os
+import socket
+from itertools import islice
+
+import dns
+import dns.query
+import dns.tsigkeyring
+import requests
+
+host_ip = str(socket.gethostbyname("magrathea.aachalon.de"))
```

Comment:
rather than hardcoding specific hostname as the AXFR target, better read it dynamically from the current SOA of the target zone

Reply:
That's a good idea!